src/templates/productive-time-card.js

```diff
@@ -1,8 +1,9 @@
 const Card = require('./card');
 const d3 = require('d3');
 
-function createProductiveCard(chartData, theme) {
-    const card = new Card('Commits per day hour (UTC)', 340, 200, theme);
+function createProductiveCard(chartData, theme, utcOffset) {
+    const title='Commits (UTC' + utcOffset + ')'
```

Comment:
a user facial change here, `Commits per day hour (UTC)` -> `Commits (UTC+8)`, I think the chart itself is already clearly   enough about what it releted to .

Reply:
good cache.